tools/webpack.assets.json

```diff
@@ -3,6 +3,10 @@
     "api": "./static/js/portico/api.js",
     "katex": ["./node_modules/katex/dist/katex.min.js"],
     "landing-page": "./static/js/portico/landing-page.js",
+    "stats": [
+              "./static/js/stats/stats.js",
+              "./node_modules/plotly.js/dist/plotly-basic.min.js"
```

Comment:
@timabbott We use a regex to check if a file is minified version or not in webpack.config.js https://github.com/zulip/zulip/blob/master/tools/webpack.config.js#L30. If it is a minified version we use script-loader for it.